federation/cluster/common.sh

```diff
@@ -165,7 +163,7 @@ function create-federation-api-objects {
     # controller manager can use to talk to the federation-apiserver.
     # Note that the file name should be "kubeconfig" so that the secret key gets the same name.
     KUBECONFIG_DIR=$(dirname ${KUBECONFIG:-$DEFAULT_KUBECONFIG})
-    CONTEXT=federation-cluster \
+    CONTEXT=${FEDERATION_NAME} \
```

Comment:
Should this be `FEDERATION_KUBE_CONTEXT` instead? See https://github.com/kubernetes/kubernetes/pull/37215/commits/355c6e17532b4d884ed7c9c8fdb08f9913db00cd#diff-8ccc33515372893705ef5e7894c585dcR131